manifests/board-image/bianbu-sd-desktop/2.1.1.toml

```diff
@@ -0,0 +1,33 @@
+format = "v1"
+
+[metadata]
+desc = "Official bianbu desktop image version 2.1.1"
+vendor = { name = "bpi-f3", eula = "" }
+upstream_version = "2.1.1"
+
+[[metadata.service_level]]
+level = "good"
+
+[[distfiles]]
+name = "bianbu-24.04-desktop-k1-v2.1.1-release-20250305144026.img.zip"
```

Comment:
So this is still `spacemit-k1`-specific. Thus the package name should reflect this and get changed to `bianbu-desktop-sd-spacemit-k1`?

Reply:
Make it `bianbu-spacemit-k1-sd-desktop`